packages/react/src/generators/application/files/common-vite/public/hello.md

```diff
@@ -0,0 +1 @@
+# Hello World
```

Comment:
Rename this file to `.gitkeep` to make sure the folder exists, and user can remove it later when they add more files.

It can be an empty file.